src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/GATKSVVCFHeaderLines.java

```diff
@@ -1,4 +1,4 @@
-package org.broadinstitute.hellbender.tools.spark.sv;
+package org.broadinstitute.hellbender.tools.spark.sv.discovery;
```

Comment:
It seems like you could also make an argument for putting this in `utils`, since probably eventually multiple tools might want to use it and read and write sv vcf files.